DeepSpeech.ipynb

```diff
@@ -446,7 +446,7 @@
     "    lstm_bw_cell = tf.nn.rnn_cell.BasicLSTMCell(n_cell_dim, forget_bias=1.0)\n",
     "    \n",
     "    # Split data because rnn cell needs a list of inputs for the BRNN inner loop\n",
-    "    layer_3 = tf.split(0, n_steps, layer_3)\n",
```

Comment:
So layer_3 has shape (18624, 988) when I run it and n_steps is 291 - 18624 / 291 = 64, which is the batch_size, as expected. Assuming we can work with a 3d tensor instead of a list of 2d tensors, couldn't we just reshape layer_3 and use `bidirectional_dynamic_rnn`? That's what I'm trying now, though it's complaining 'None values not supported', so I've either made a bad/incorrect assumption or gotten the parameters wrong...